src/it/scala/org/zalando/spearheads/innkeeper/routes/AcceptanceSpecsHelper.scala

```diff
@@ -143,9 +149,18 @@ object AcceptanceSpecsHelper extends ScalaFutures with Matchers {
   def getUpdatedRoutes(localDateTime: LocalDateTime, token: String): HttpResponse =
     getUpdatedRoutes(RequestParameters.urlDateTimeFormatter.format(localDateTime), token)
 
+  private def doDelete(requestUri: String, token: String = ""): HttpResponse = {
```

Comment:
for one line functions, we can remove brackets, like we did in the same file, before